File: app/src/main/java/ninja/eigenein/nicknames/activities/BaseActivity.java
package ninja.eigenein.nicknames.activities;

import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import ninja.eigenein.nicknames.R;

public abstract class BaseActivity extends AppCompatActivity {

    protected void setupToolbar() {
        final Toolbar toolbar = (Toolbar)findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        final ActionBar actionBar = getSupportActionBar();
        actionBar.setDisplayHomeAsUpEnabled(true);
    }
}

File: app/src/main/kotlin/ninja/eigenein/nicknames/activities/BaseActivity.kt
package ninja.eigenein.nicknames.activities

import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.Toolbar
import ninja.eigenein.nicknames.R

public abstract class BaseActivity : AppCompatActivity() {

    protected fun setupToolbar() {
        val toolbar = findViewById(R.id.toolbar) as Toolbar
        setSupportActionBar(toolbar)

        val actionBar = getSupportActionBar()
        actionBar.setDisplayHomeAsUpEnabled(true)
    }
}
